pkgs/top-level/all-packages.nix

```diff
@@ -34063,8 +34063,6 @@ with pkgs;
 
   ninjas2 = callPackage ../applications/audio/ninjas2 { };
 
-  nncp = darwin.apple_sdk_11_0.callPackage ../tools/misc/nncp { };
```

Comment:
This removal does not look like it was intended.

Reply:
That was intended, packages are moving to `pkgs/by-name` and if it breaks darwin then that is an apple fanboy problem.

Reply:
Ah. Didn't catch that. But should we really break macOS on purpose? This apple_sdk_11_0 is needed for buildGoPackage to work. There is no other strong incentive just now to move to `pkgs/by-name` otherwise.

Reply:
meta already has `    broken = stdenv.isDarwin;` btw